apisupport/apisupport.ant/test/unit/src/org/netbeans/modules/apisupport/project/ui/wizard/common/CreatedModifiedFilesTest.java

```diff
@@ -222,7 +223,7 @@ public void testAddLookupRegistration() throws Exception {
         assertNotNull(registry);
         InputStream is = registry.getInputStream();
         try {
-            BufferedReader r = new BufferedReader(new InputStreamReader(is, "UTF-8"));
+            BufferedReader r = new BufferedReader(new InputStreamReader(is, StandardCharsets.UTF_8));
```

Comment:
how about try-with-resources here?

Reply:
this PR does only one inspection. It would be too much at once otherwise and would make review very tiresome.